chapter02/qli/src/main/java/spring/training/model/User.java

```diff
@@ -0,0 +1,23 @@
+package spring.training.model;
+
+public class User {
```

Comment:
yeah, probably sure for the username, but still User will need to be able to update their passwords maybe even every 3 month like us, lol